client/src/components/classesPage/ClassCreateView.js

```diff
@@ -133,20 +135,122 @@ function ClassCreateView(props) {
+          // setTimeout(() => {
+          //   campaignData.campaign_id = validated.campaign_id; // tacking on for submitCD
+          //   submitClassData(
+          //     listData,
+          //     validated.list_ids[0],
+          //     recipientData,
+          //     campaignData
+          //   );
+          // }, 30000);
         }
       })
       .catch(err => console.log(err));
   };
 
+  // Schedule campaign for 2 weeks after class date
+  const sendAllToSendgrid2 = () => {
+    addList(listData.classnameInput + " 2wk")
+      .then(res => {
+        validated.list_ids.push(res.data.id);
+        console.log(`160`);
+        console.log(validated.list_ids[1]);
+        console.log(validated.recipient_ids);
+        return addContacts(validated.list_ids[1], validated.recipient_ids);
+      })
+      .then(res => {
+        if (res.status === 201) {
+          console.log(`167`);
+          console.log(validated.list_ids[0]);
+          console.log(validated.recipient_ids);
+          return addRefreshr(newRefreshr);
+        }
+      })
+      .then(res => {
+        if (res.status === 201) {
+          validated.campaign_id.push(res.data.id);
+          console.log(`174`);
+          console.log(timeTriData[1]);
+          console.log(validated.campaign_id[1]);
+          return scheduleRefreshr(timeTriData[1], validated.campaign_id[1]);
+        }
+      })
+      .then(res => {
+        if (res.status === 201) {
+          validated.schedule_code = res.status;
+          console.log(
+            `Success! Your campaign ${res.data.id} is scheduled for ${
+            res.data.send_at
+            }. Status is "${res.data.status}"!`
+          );
+          // setTimeout(() => {
+          sendAllToSendgrid3()
+          // }, 2000);
+          // setTimeout(() => {
+          //   campaignData.campaign_id = validated.campaign_id; // tacking on for submitCD
+          //   submitClassData(
+          //     listData,
+          //     validated.list_ids[0],
+          //     recipientData,
+          //     campaignData
+          //   );
+          // }, 30000);
+        }
+      })
+      .catch(err => console.log(err));
+  }
+
+  // Schedule campaign for 2 months after class date
+  const sendAllToSendgrid3 = () => {
+    addList(listData.classnameInput + " 2mo")
+      .then(res => {
+        validated.list_ids.push(res.data.id);
+        console.log(`212`);
+        console.log(validated.list_ids[2]);
+        console.log(validated.recipient_ids);
+        return addContacts(validated.list_ids[2], validated.recipient_ids);
+      })
+      .then(res => {
+        if (res.status === 201) {
+          console.log(`219`);
+          return addRefreshr(newRefreshr);
+        }
+      })
+      .then(res => {
+        if (res.status === 201) {
+          validated.campaign_id.push(res.data.id);
+          console.log(`226`);
+          console.log(timeTriData[2]);
+          console.log(validated.campaign_id[2]);
```

Comment:
Still tons of console.logs but we don't need to remove them until next week.

Reply:
Yea, was planning to after we square away with saving to the DB.